<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>formRegister</class>
    <widget class="QDialog" name="formRegister">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>745</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Registro de usuario</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="labelTitle">
            <property name="geometry">
                <rect>
                    <x>190</x>
                    <y>20</y>
                    <width>291</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>18</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(33, 37, 41);</string>
            </property>
            <property name="text">
                <string>Registro de usuario</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblFirstName">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>250</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Nombre</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtFirstName">
            <property name="geometry">
                <rect>
                    <x>59</x>
                    <y>290</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblLastName">
            <property name="geometry">
                <rect>
                    <x>339</x>
                    <y>250</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Apellido</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtLastName">
            <property name="geometry">
                <rect>
                    <x>349</x>
                    <y>290</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblUsername">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>350</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Usuario</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtUsername">
            <property name="geometry">
                <rect>
                    <x>59</x>
                    <y>380</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblPassword">
            <property name="geometry">
                <rect>
                    <x>339</x>
                    <y>350</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Contrase√±a</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtPassword">
            <property name="geometry">
                <rect>
                    <x>349</x>
                    <y>380</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="echoMode">
                <enum>QLineEdit::Password</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblEmail">
            <property name="geometry">
                <rect>
                    <x>59</x>
                    <y>530</y>
                    <width>231</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Email</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtEmail">
            <property name="geometry">
                <rect>
                    <x>59</x>
                    <y>560</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblDocumentType">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>440</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Tipo de documento</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QComboBox" name="txtDocumentType">
            <property name="geometry">
                <rect>
                    <x>59</x>
                    <y>470</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
        </widget>
        <widget class="QLabel" name="lblDocumentNumber">
            <property name="geometry">
                <rect>
                    <x>340</x>
                    <y>440</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>N√∫mero de documento</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtDocumentNumber">
            <property name="geometry">
                <rect>
                    <x>349</x>
                    <y>470</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="logo">
            <property name="geometry">
                <rect>
                    <x>230</x>
                    <y>90</y>
                    <width>191</width>
                    <height>131</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: none;</string>
            </property>
            <property name="pixmap">
                <pixmap>../assets/logo_optica.png</pixmap>
            </property>
            <property name="scaledContents">
                <bool>true</bool>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblError">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>690</y>
                    <width>371</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(255, 85, 0);</string>
            </property>
            <property name="text">
                <string>Error</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" name="btnRegister">
            <property name="geometry">
                <rect>
                    <x>250</x>
                    <y>630</y>
                    <width>171</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    background-color: rgb(111, 166, 80);
                    color: rgb(255, 255, 255);
                    border-radius: 8px;
                </string>
            </property>
            <property name="text">
                <string>Registrar</string>
            </property>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>